ansible/roles/awscli/tasks/main.yml

```diff
@@ -0,0 +1,31 @@
+---
+- name: Make sure that pip is installed
+  apt: name=python-pip state=installed
+  tags: 
+    - awscli
```

Comment:
In some of our other roles the same thing is done.  I'm not sure it's 100% needed, but at the same time, it's not going to stop it from working.  We should maybe figure out what's the best way to do tags and then stick to that.